parser/openfoodfacts_taxonomy_parser/test.txt

```diff
@@ -0,0 +1,37 @@
+# test taxonomy
```

Comment:
why do we have this file here ?

Reply:
I also had the same doubt since I wasn't able to run this module while inside the package. This "test.txt" file should be outside `openfoodfacts_taxonomy_parser` I think.

Reply:
It's to test the parser myself when I'm writing it

Reply:
ok, but maybe do not commit that ;-)

Reply:
Do I alse remove the end (if __name__== ...)  ?